shared/desktop/app/app-state.js

```diff
@@ -4,7 +4,6 @@ import {app, screen} from 'electron'
 import fs from 'fs'
 import path from 'path'
 import {appBundlePath} from './paths'
-import jsonfile from 'jsonfile'
```

Comment:
Instead of pulling in this dependency lets just use the builtin stuff. its not complex enough to warrant it